### ISSUE TITLE
Fabric Config Files not written or used

### ISSUE THREAD

**Issue by Knito58** [WiIIiam278/HuskHomes#512]
Fabric Version: 0.14.23-1.20.2
Huskhomes: Huskhomes-Fabric-4.5.3.jar
MC: 1.20.2

Playing as survival single player (SSP).

No config files get written (or I'm too dumb to find em). If I follow the webpage instructions and generate the config files by hand - everything there is really well documented - they seem not to become used by the mod. (no tables generated in my mysql db)

No error is thrown. The commands do not show up in the command input line (stay red).

The keyword "husk" or "home" cannot be found in latest.log. (I cannot find it....)

Thank you for investigating that or put me on the track.

**Comment by WiIIiam278** [comment]
HuskHomes is a dedicated server mod :) 

Thanks

**Comment by Knito58** [comment]
I believed that my local single player installation when running is split into a server - client model. Therefore I installed a mariadb on "localhost". So - no way?

**Comment by WiIIiam278** [comment]
yeah, sorry - this only runs and is intended for dedicated servers - not the internal server :) sorry bout that!
